src/model/OrionOffchainCursor.ts

```diff
@@ -0,0 +1,20 @@
+import { Entity, Column, PrimaryColumn } from 'typeorm'
+
+@Entity()
```

Comment:
I think it's better to create this in admin schema
```suggestion
@Entity({ schema: 'admin' })
```